Gruntfile.js

```diff
@@ -1,70 +1,64 @@
 module.exports = function(grunt) {
 
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
-			files: ['Gruntfile.js', 'src/**/*.js', 'tests/**/*.js'],
-			options: {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    // Code formatting / linting
+    jscs: {
+      src: 'src/hello.js',
```

Comment:
Applying to one file for now - will add the others (including tests) if/when happy with changes.